src/modules/Support/html_client/mod_support_ticket.html.twig

```diff
@@ -4,199 +4,213 @@
-                        <td><a href="{{ 'order/service/manage'|link }}/{{ ticket.rel_id }}">{{ 'Service #'|trans }} {{ ticket.rel_id }}</a></td>
-                    </tr>
-                    {% endif %}
-                    </tbody>
-
-                </table>
-
-            </section>
-        </div>
-    </article>
-    <article class="span8 data-block">
-        <div class="data-container">
-            <header>
-                <h2>{{ 'Last tickets'|trans }}</h2>
-                <div class="data-header-actions">
-                    <ul>
-                        <li>
-                            <a class="btn btn-small" href="{{ '/support'|link }}">{{ 'Back to tickets list'|trans }}</a>
-                        </li>
-                    </ul>
                 </div>
-            </header>
-            <section>
-                <table class="table table-bordered table-condensed">
-                    <thead>
-                    <tr>
-                        <th>{{ 'Subject'|trans }}</th>
-                        <th>{{ 'Help desk'|trans }}</th>
-                        <th>{{ 'Status'|trans }}</th>
-                        <th>{{ 'Submitted'|trans }}</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    {% set tickets = client.support_ticket_get_list({ "per_page": 5, "page": request.page }) %}
-                    {% for i, t in tickets.list %}
-                    <tr {% if ticket.id == t.id %}class="alert alert-info"{% endif %}>
-                        <td><a href="{{ 'support/ticket'|link }}/{{t.id}}">{{ t.subject|truncate(40) }}</a></td>
-                        <td>{{ t.helpdesk.name }}</td>
-                        <td>
-                            <span class="label {% if t.status=='open' %}label-info{% elseif t.status == 'on_hold' %}label-warning{% endif %}">{{ mf.status_name(t.status) }}</span>
-                        </td>
-                        <td>{{ t.created_at|timeago }} {{ 'ago'|trans }}</td>
-                    </tr>
-                    {% else %}
-                    <tr>
-                        <td colspan="5">{{ 'The list is empty'|trans }}</td>
-                    </tr>
-                    {% endfor %}
-
-                    </tbody>
-                </table>
-            </section>
+                <div class="card-body">
+                    <table class="table table-hover table-sm">
+                        <tbody>
+                        <tr>
+                            <td>{{ 'Help desk'|trans }}</td>
+                            <td>{{ ticket.helpdesk.name }}</td>
+                        </tr>
+
+                        <tr>
+                            <td>{{ 'Status'|trans }}</td>
+                            <td>
+                                <div class="badge {% if ticket.status=='open' %}bg-success{% elseif ticket.status == 'on_hold' %}bg-warning{% endif %}">{{ mf.status_name(ticket.status) }}</div>
```

Comment:
The status for closed is not correctly represented
![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/d464f111-c698-4c15-aa1f-496e88b07a1d)
![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/434b0063-0612-47e0-89ba-c34c2394cfc1)